vmaas/webapp/repos.py

```diff
@@ -12,7 +14,10 @@
     REPO_THIRD_PARTY
 )
 from vmaas.common.date_utils import format_datetime
-from vmaas.common.webapp_utils import paginate, none2empty, parse_datetime, filter_item_if_exists, try_expand_by_regex
+from vmaas.common.webapp_utils import paginate, none2empty, parse_datetime, \
+                  filter_item_if_exists, try_expand_by_regex, strip_prefixes
+
+REPO_PREFIXES = getenv("REPO_NAME_PREFIXES", "").split(",")
```

Comment:
Add it to webapp/app.py. Do not load it in all files (repos.py, updated.py, vulnerabilities.py), just import it. Check how other vars are used. Do not import getenv directly. Use os.getenv as used in other places.